editor/editor_settings.cpp

```diff
@@ -1483,17 +1483,22 @@ Ref<Shortcut> EditorSettings::get_shortcut(const String &p_name) const {
 	// Use the first item in the action list for the shortcut event, since a shortcut can only have 1 linked event.
 
 	Ref<Shortcut> sc;
+	bool no_shortcut = false;
 	const Map<String, List<Ref<InputEvent>>>::Element *builtin_override = builtin_action_overrides.find(p_name);
 	if (builtin_override) {
-		sc.instantiate();
-		sc->set_event(builtin_override->get().front()->get());
-		sc->set_name(InputMap::get_singleton()->get_builtin_display_name(p_name));
+		if (!builtin_override->get().is_empty()) {
+			sc.instantiate();
+			sc->set_event(builtin_override->get().front()->get());
+			sc->set_name(InputMap::get_singleton()->get_builtin_display_name(p_name));
+		} else {
+			no_shortcut = true;
+		}
 	}
 
 	// If there was no override, check the default builtins to see if it has an InputEvent for the provided name.
 	if (sc.is_null()) {
 		const OrderedHashMap<String, List<Ref<InputEvent>>>::ConstElement builtin_default = InputMap::get_singleton()->get_builtins().find(p_name);
-		if (builtin_default) {
+		if (builtin_default && !no_shortcut) {
```

Comment:
This `no_shortcut` is still referring to the `builtin_override` value, while this if statement is used for the `builtin_default`. `builtin_default` should be checked for empty in this case, as its value is what is accessed in the if block.
```suggestion
		if (builtin_default && !builtin_default.get().is_empty()) {
```